dataset/tags.json5

```
@@ -32,21 +32,21 @@
   "jarilo-6": {
     en: "Jarilo-Ⅵ",
     ja: "ヤリーロ-Ⅵ",
-    "zh-CN": "",
+    "zh-CN": "雅利洛-Ⅵ",
     title: {
       en: "Chinese & Japanese translations for words related to Jarilo-Ⅵ",
       ja: "ヤリーロ-Ⅵに関する言葉の英語・中国語表記一覧",
-      "zh-CN": "关于**的词语的英语和日语翻译",
+      "zh-CN": "关于雅利洛-Ⅵ的词语的英语和日语翻译",
     },
   },
   luofu: {
     en: "The Xianzhou Luofu",
     ja: "仙舟・羅浮",
```

Because the notations of The Xianzhou Luofu are not consistent in Japanese and Chinese (仙舟・羅浮 / 仙舟: 罗浮 vs 仙舟「羅浮」 / 仙舟「罗浮」), let's use the notation with the brackets in Japanese too if we use that in Chinese.

```suggestion
    ja: "仙舟「羅浮」",
```